src/index.js

```diff
@@ -92,6 +92,9 @@ export function getExternalScripts(scripts, fetch = defaultFetch, errorCallback
 			}
 
 			return response.text();
+		}).catch(e => {
+			errorCallback();
+			throw new Error(`${scriptUrl} load failed (${e.name}: ${e.message})`);
```

Comment:
```suggestion
			throw e;
```

error 直接原样 throw，从而保留原来的异常栈信息